backend/core/models/user_daily_usage.py

```diff
@@ -0,0 +1,54 @@
+from logger import get_logger
+from models.databases.supabase.supabase import SupabaseDB
+from models.settings import get_supabase_db
+from models.user_identity import UserIdentity
+
+logger = get_logger(__name__)
+
+
+class UserDailyUsage(UserIdentity):
```

Comment:
Rename into UserUsage so it doesn't imply daily

Reply:
It is daily though

Reply:
The key is user_id X date of the day

Reply:
Should I rename it UserUsage with an attribute like today_requests_count ?

Reply:
Yes ! So that later on we can use it for all kind of rate limiting ;) WDYT ?

Reply:
I already made the change after your comment but after reflecting on it, I think we should not name our variables nor files based on potential future feature considerations. 
The naming was very clear, now it's only more confusing :/